srcs/runners_manager/vm_creation/aws/AwsManager.py

```diff
@@ -62,15 +62,29 @@ def create_vm(
                 {
                     'Key': 'Name',
                     'Value': runner.name
+                },
+                {
+                    'Key': 'lifecycle_autostop',
+                    'Value': 'no'
+                },
+                {
+                    'Key': 'lifecycle_autostart',
+                    'Value': 'no'
+                },
+                {
+                    'Key': 'map-migrated',
+                    'Value': 'mig42992'
+                },
+                {
+                    'Key': 'owner',
+                    'Value': 'ci'
+                },
+                {
+                    'Key': 'tool',
+                    'Value': 'runner-manager'
                 }
             ]
 
-            for key, value in self.tags.items():
```

Comment:
You have some suspicion the `self.tags` dict was not properly filled?
Because I'm OK with your changes, but it's really basic and I'm wondering what would prevent the original code from working fine.